### ISSUE TITLE
Blisters or bullae on tympanic membrane

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10039]
**Preferred term label:**
Blisters or bullae on tympanic membrane

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0040090

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Bullous myringitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I think the standard name for this is Bullous Myringitis
https://www.ncbi.nlm.nih.gov/books/NBK553029/

**Comment by MickeySegal** [comment]
That was the disease that I included in the original new term request, but I've also added it as a synonym for the finding.  But I don't see an HPO term.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000422
- label: Tympanic membrane bulla
- parents: [Abnormal tympanic membrane morphology (HP:0040090)](https://hpo.jax.org/app/browse/term/HP:0040090)
- pmid: PMID:31971705
- def: Presence of blisters (bullae) on the tympanic membrane, often accompanied by fever, thickening, and erythematous appearance of the tympanic membrane, decreased or absent light reflex, and decreased mobility.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10039
- synonyms: Tympanic membrane blister(NONE)|Bullous myringitis(NONE)
